### PR TITLE
feat(xapi-object-to-xo/host): patchesRequiringReboot

### PR DESCRIPTION
**Pull request by pdonias**
See vatesfr/xo-web#2124

### CODE REVIEW

File: src/xapi-object-to-xo.js [hunk 1]
```diff
@@ -195,6 +195,7 @@ const TRANSFORMS = {
         }),
       agentStartTime: toTimestamp(otherConfig.agent_start_time),
       tags: obj.tags,
+      patchesRequiringReboot: link(obj, 'updates_requiring_reboot'),
```
Comment (julien-f) [inline src/xapi-object-to-xo.js]: Shouldn't we call it `updatesRequiringReboot`?

Because IMHO it concerns more than just patches.

Comment (pdonias) [inline src/xapi-object-to-xo.js]: True. But should xo-web be aware of the difference between patch and update?

Comment (olivierlambert) [inline src/xapi-object-to-xo.js]: I think it's far too early to decide on this right now. We should merge this and change in the future if necessary.

Comment (julien-f) [inline src/xapi-object-to-xo.js]: `rebootRequired: !isEmpty(obj.updates_requiring_reboot)`.

**Pull request merged**